livekit-ffi/protocol/track.proto

```diff
@@ -7,21 +7,10 @@ import "handle.proto";
 import "video_frame.proto";
 import "audio_frame.proto";
 
-message VideoCaptureOptions {
-  VideoResolution resolution = 1;
-}
-
-message AudioCaptureOptions {
-  bool echo_cancellation = 1;
-  bool noise_suppression = 2;
-  bool auto_gain_control = 3;
-}
-
 // Create a new VideoTrack from a VideoSource
 message CreateVideoTrackRequest {
   string name = 1;
```

Comment:
is this a required field? can a user leave it blank?

Reply:
It is not required, by the FFI server, it will depends on the client implementations. At the moment, the name can be blank. Maybe it is better to default it to smth like `unnamed track`?